src/index.html

```diff
@@ -7,6 +7,427 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="link-img">
+        <img
+          src="./images/logo.png"
+          class="logo"
+          alt="0"
+        >
```

Comment:
Check the BEM structure here. Look at my suggestion 
```suggestion
    <header class="header">
      <a href="#home" class="header__logo logo">
        <img
          src="./images/logo.png"
          class="logo__image"
          alt="0"
        >
```